src/components/icons/non-selected-pin.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const NonSelectedPinIcon = ({ className }) => (
+  <svg className={className} viewBox="0 0 24 24">
+    <path
+      d="M12 4a8 8 0 1 1 0 16 8 8 0 0 1 0-16zm0 2a6 6 0 1 0 0 12 6 6 0 0 0 0-12zm0 4a2 2 0 1 1 0 4 2 2 0 0 1 0-4z"
+      fill="#FFF"
+      fillRule="nonzero"
+      fillOpacity=".55"
```

Comment:
From what I've seen it seems like we try to leave stying off icons and handle that with class names. This is why the colors aren't changing when I toggle the theme.